.readthedocs.yaml

```diff
@@ -1,11 +1,18 @@
 # Required by Read The Docs
 version: 2
 
-formats: all
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.8"
 
 python:
-  version: "3.8"
+  install:
+    - requirements: doc/userguide/requirements.txt
```

Comment:
Was, this (the second commit) required to make things work? Or a stylistic change?

Reply:
Required. RtD was failing to find the correct sphinx theme without this. There seem to be some ways of making this work, this was the one I was able to understand and follow.

Reply:
(cf https://readthedocs.org/projects/suri-rtd-test/builds/22135861/ )